MediaBrowser.XbmcMetadata/Providers/BaseNfoProvider.cs

```diff
@@ -72,7 +73,7 @@ public bool HasChanged(BaseItem item, IDirectoryService directoryService)
                 return false;
             }
 
-            return file.Exists && _fileSystem.GetLastWriteTimeUtc(file) > item.DateLastSaved;
+            return file.Exists && item.DateLastSaved != DateTimeOffset.MinValue && _fileSystem.GetLastWriteTimeUtc(file) > item.DateLastSaved;
```

Comment:
Is this still relevant?

Reply:
Since this happens after we transfer to a DTO, it should still be relevane. @theguymadmax maybe you can check again?